specification/communication/data-plane/JobRouter/preview/2021-10-20-preview2/communicationservicejobrouter.json

```diff
@@ -1052,6 +1052,56 @@
         }
       }
     },
+    "/routing/jobs/{jobId}/assignments/{assignmentId}:unassign": {
```

Comment:
The rest of the spec refers to the ID of the job as {id} whereas here it's {jobId}, Might be worth having a consistent name.

Reply:
We named this jobId because there are two ids in this route. One jobId and the other one assignmentId.